src/components/ResourceList/ResourceList.tsx

```diff
@@ -431,9 +432,9 @@ export class ResourceList extends React.Component<CombinedProps, State> {
     ) : null;
 
     const headerMarkup = (showHeader || needsHeader) &&
-      listNode && (
+      this.listRef.current && (
```

Comment:
I thought `listNode` had been changed to the new ref syntax quite a while ago? 😕